minio/api.py

```diff
@@ -1949,6 +1949,54 @@ def _get_bucket_location(self, bucket_name):
             return 'eu-west-1'
         return location
 
+    def _get_assume_role_creds(self, role_arn=None, role_session_name=None, policy=None, duration_seconds=None):
```

Comment:
1. Make this method public.
2. `role_*` is too verbose. you could have `arn` and `session_name`
3. I think `duration` is good enough than `duration_seconds`
4. Avoid `'` usage instead use `"` which is more standard.